hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBuffer.java

```diff
@@ -353,7 +357,7 @@ public void testS3SecretCacheSizePostDoubleBufferFlush() throws IOException {
 
     // Flush the current buffer.
     doubleBuffer.flushCurrentBuffer();
-
+    cache = secretManager.cache();
```

Comment:
This part of the change seems to be unnecessary, the object returned by `cache()` is `final`, does not change.

Reply:
Done!